src/bot/lib/utils/only.js

```diff
@@ -1,9 +1,63 @@
-export function only(channel, modifier) {
-  return (incommingMessage, messageContext) => {
-    if (messageContext.channel === channel) {
-      return messageContext
+export function onlyFor(channel, modifier) {
+  const memoryKey = `${Math.random().toString(36).substring(2)}-${channel}-scoped-memory`
```

Comment:
Is there any case where two channel has the same name ?

Reply:
It shouldn't be possible 🤷🏾‍♂️

Reply:
But this is mostly to avoid conflict with other modifiers that are not scoped by the `only` function. For whatever reason they could decide to save a data about a user which is also the broadcaster and the channel name.